parkings/tests/api/enforcement/test_activate_permit_series.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+
+from django.urls import reverse
+from django.utils import timezone
+from rest_framework.status import HTTP_200_OK
+
+from parkings.models import PermitSeries
+
+
+def get_url(obj):
+    return reverse('enforcement:v1:permitseries-detail', kwargs={'pk': obj.pk})
+
+
+def test_activate_permit_series(staff_api_client):
+
+    new_permit_series = PermitSeries.objects.create(active=False)
+    old_permit_series = PermitSeries.objects.create(active=True)
+    old_permit_series.created_at = timezone.now() - datetime.timedelta(3)
+    old_permit_series.save()
+
+    assert PermitSeries.objects.all().count() == 2
+    response = staff_api_client.post(get_url(new_permit_series) + 'activate/')
+    assert response.data['status'] == "OK"
+    assert response.status_code == HTTP_200_OK
+    assert PermitSeries.objects.latest_active() == new_permit_series
+    assert PermitSeries.objects.all().count() == 1
```

Comment:
Maybe this could be split to two test cases instead? (one checking that the activation works, i.e. deactivates other series and activates the one and another test that checks that the old series are deleted)

Reply:
...and since `activate_permit_series` is already part of the test filename, the function names don't have to have that as part of their name. The tests could be named e.g. `test_series_gets_activated` and `test_old_series_are_pruned`.